tools/webpack/blocks.js

```diff
@@ -220,15 +220,21 @@ module.exports = [
 		].filter( Boolean ),
 	},
 	{
+		...baseConfig,
```

Comment:
This bring some shared setting used for all other configs:
- Browserslist integration.
- Bundle analyzer integration.
- Custom config for the terser plugin.
- Development vs production mode aligned with how `npm run build` and `npm run dev` work.
- Integration with devtol.